bpf/unwinders/native.bpf.c

```diff
@@ -240,6 +240,23 @@ typedef struct {
     stack_unwind_row_t rows[MAX_UNWIND_TABLE_SIZE];
 } stack_unwind_table_t;
 
+typedef struct {
+    u32 pc_not_covered;
+    u32 no_unwind_info;
+    u32 missed_filter;
+    u32 mapping_not_found;
+    u32 chunk_not_found;
+    u32 null_unwind_table;
+    u32 table_not_found;
+    u32 rbp_failed;
```

Comment:
This name probably shouldn't be architecture specific.

Reply:
I wanted to keep things as close as possible to the log messages / comments in the unwinder. We can clean up all of them at the same time in a separate PR, I think.